tce/opset/base/Makefile.am

```diff
@@ -2,18 +2,21 @@ basedir = ${prefix}/share/tce/opset/base
 avalondir = ${prefix}/share/tce/opset/base
 simple_iodir = ${prefix}/share/tce/opset/base
 doubledir = ${prefix}/share/tce/opset/base
+implicit_lsudir = ${prefix}/share/tce/opset/base
```

Comment:
Strictly put LSU is for load-store unit. Here you are defining load-store _operations_ (that can be in function units). Implicit is also too generic word, what you are doing is postincrement_load_store, right?

Reply:
But perhaps we want to add postdecrement too in the future so "implicit_load_store" or such?

Reply:
sure

Reply:
fixed